index.html

```diff
@@ -347,13 +341,13 @@ <h3>Locales and Internationalization</h3>
+        <p>Specifications for document formats or protocols usually define the exchange, processing, or display of various data values or data structures. The Web primarily relies on text files for the serialization and exchange of data: even raw bytes are usually transmitted using a string serialization such as base64. Thus <a>non-linguistic fields</a> on the Web are also normally made up of strings. The important distinction here is that <a>non-linguistic fields</a> are generally interpreted by or meant for consumption by the underlying application, rather than by a user.</p>
         
-        <p class="definition"><dfn data-lt="locale-neutral|locale neutral">Locale-neutral</dfn>. A <a>non-linguistic field</a> is said to be <a>locale-neutral</a> when it is stored or exchanged in a format that is not specifically appropriate any given language, locale, or culture and which can be interpreted unambiguously for presentation in a <a>locale aware</a> way.</p>
+        <p class="definition"><dfn data-lt="locale-neutral|locale neutral">Locale-neutral</dfn>. A <a>non-linguistic field</a> is said to be <a>locale-neutral</a> when it is stored or exchanged in a format that is not specifically appropriate for any given language, locale, or culture and which can be interpreted unambiguously for presentation in a <a>locale aware</a> way.</p>
         
         <p>Many specifications use a serialization scheme, such as those provided by [[XMLSCHEMA11-2]] or [[JSON-LD]], to provide a <a>locale neutral</a> encoding of <a>non-linguistic fields</a> in document formats or protocols.</p>
         
-        <p>A <a>locale-neutral</a> representation might itself be linked to a specific cultural preference, but such linkages should be minimized. An example of this are the ISO8601 serializations of date/time values. Many of these are linked to the Gregorian calendar, but the format, field order, separators, and visual appearance are not specifically suitable to any locale (they are intended to be machine readable) and, as shown in the <a href="#example-locale-variation">example</a> above, the value can be converted for display into any calendar or locale.</p>
+        <p>A <a>locale-neutral</a> representation might itself be linked to a specific cultural preference, but such linkages should be minimized. For example, many of the ISO8601  date/time value serialisations are linked to the Gregorian calendar, but the format, field order, separators, and visual appearance are not specifically suitable to any locale (they are intended to be machine readable) and, as shown in the <a href="#example-locale-variation">example</a> above, the value can be converted for display into any calendar or locale.</p>
```

Comment:
British spelling of `serialization`.
Extra space after ISO8601.

Reply:
Ok, i'll fix these things early morning my time tomorrow.  Should i merge after ?

Reply:
Please do!